lib/utils/types.js

```diff
@@ -130,6 +131,16 @@ function isClassDeclaration(node) {
   return node !== undefined && node.type === 'ClassDeclaration';
 }
 
+/**
+ * Check whether or not a node is a ClassExpression.
+ *
+ * @param {Object} node The node to check.
+ * @returns {boolean} Whether or not the node is a ClassExpression.
+ */
+function isClassExpression(node) {
```

Comment:
We probably want to avoid adding this helper. I'm not a fan of these trivial type check helpers and plan to make them deprecated.

Reply:
I wasn't sure. These helpers are pretty trival, but I think it'd be weirder to have the call in `isEmberCoreModule` be `(types.isClassDeclaration(node) || node.type === 'ClassExpression')` or to do more major refactors to that code.

Reply:
I'm deprecating these helpers here: https://github.com/ember-cli/eslint-plugin-ember/pull/1720

It's fine if we're slightly inconsistent with the type checks while we're migrating away from them, but I do want avoid adding new trivial type check helpers.